### PR TITLE
Feature/eligibility existing claimants

### PR DESCRIPTION
**Pull request by mikeSmith08**

### CODE REVIEW

File: api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityService.java [hunk 1]
```diff
@@ -18,10 +18,11 @@
...
     /**
-     * Determines the eligibility for a given status. If the claimant's nino is not found in the database,
+     * Determines the eligibility for a given claimant. If the claimant's NINO is not found in the database,
      * the external eligibility service is called.
+     *
      * @param claimant the claimant to check the eligibility for
-     * @return an eligibility response for the claimant
+     * @return the eligibility response for the claimant
      */
     public EligibilityResponse determineEligibility(Claimant claimant) {
```
Comment (dwybourn) [inline api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityService.java]: Should we rename this method to state it's for new claims?

Comment (mikeSmith08) [reply api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityService.java]: Yep, done.

File: api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityService.java [hunk 2]
```diff
@@ -33,4 +34,15 @@ public EligibilityResponse determineEligibility(Claimant claimant) {
                 .eligibilityStatus(eligibilityStatus)
                 .build();
     }
+
+    /**
+     * Determines the eligibility for the given existing claimant. No check is made on the NINO as they already exist in the
+     * database. The eligibility status is simply checked by calling the external service.
+     *
+     * @param claimant the claimant to check the eligibility for
+     * @return an eligibility response for the claimant
```
Comment (dwybourn) [inline api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityService.java]: Should be 'the' instead of 'an'.

Comment (mikeSmith08) [reply api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityService.java]: Done.

**Pull request merged**